### PR TITLE
Implement create

### PR DESCRIPTION
**Pull request by KapStorm**
It has requiring validations

image

### CODE REVIEW

File: spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala [hunk 1]
```diff
@@ -253,7 +253,9 @@ object DatabaseTablesDAO {
       val value = body(body.keys.toList(j - i - 1))
       preparedStatement.setObject(j, value)
     }
-    val _ = preparedStatement.executeUpdate()
+    val result = preparedStatement.executeQuery()
+    result.next()
+    result.getString(1)
```
Comment (AlexITC) [inline spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala:258]: What's this returning? is it guaranteed to be present?

File: spra-play-server/src/main/scala/net/wiringbits/spra/admin/utils/QueryBuilder.scala [hunk 2]
```diff
@@ -29,6 +29,7 @@ object QueryBuilder {
       |VALUES (
       |  ${sqlValues.toString()}
       |)
+      |RETURNING $primaryKeyField::TEXT
```
Comment (AlexITC) [inline spra-play-server/src/main/scala/net/wiringbits/spra/admin/utils/QueryBuilder.scala:32]: Does this work with int pks?

File: spra-play-server/src/main/resources/application.conf [hunk 3]
```diff
@@ -6,14 +6,18 @@ dataExplorer {
...
+    users {
+      tableName = "users"
+      primaryKeyField = "user_id"
+      hiddenColumns = ["password", "email"]
+      nonEditableColumns = ["user_id", "email", "created_at", "verified_on", "name"]
+      canBeDeleted = false
+      filterableColumns = ["name", "last_name"]
+      createFilter {
+        requiredColumns = ["name", "email", "password"]
+        nonRequiredColumns = ["last_name", "discord_webhook_url"]
```
Comment (AlexITC) [inline spra-play-server/src/main/resources/application.conf:18]: do we really need both options? also, I see no tests.

**Review by AlexITC — APPROVED** [review]

File: spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala (hunk 1 above)
Comment (KapStorm) [reply spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala:258]: It's the value of the id of the table, it always returns a String (::text) value of the id. We have to return it to make known React-Admin that the row was created and React-Admin automatically redirects from the /table/create page to the /table/:id page

File: spra-play-server/src/main/resources/application.conf (hunk 3 above)
Comment (KapStorm) [reply spra-play-server/src/main/resources/application.conf:18]: It's to make known React-Admin which fields will be displayed and required in the /table/create page. The required and non required fields will be defined in the database (with DEFAULT constraint)

File: spra-play-server/src/main/scala/net/wiringbits/spra/admin/utils/QueryBuilder.scala (hunk 2 above)
Comment (KapStorm) [reply spra-play-server/src/main/scala/net/wiringbits/spra/admin/utils/QueryBuilder.scala:32]: Yes, it works

**Pull request merged**